### PR TITLE
Add OSM example to website

### PR DESCRIPTION
**Pull request by xintongxia**
#### Change List
- rename openstreets-map -> map-tile
- add map-tile to website/demos
- fix bitmap artifact

demo

### CODE REVIEW

File: modules/layers/src/bitmap-layer/bitmap-layer.js [hunk 1]
```diff
@@ -32,7 +32,9 @@ import fs from './bitmap-layer-fragment';
 const DEFAULT_TEXTURE_PARAMETERS = {
   [GL.TEXTURE_MIN_FILTER]: GL.LINEAR_MIPMAP_LINEAR,
   // GL.LINEAR is the default value but explicitly set it here
-  [GL.TEXTURE_MAG_FILTER]: GL.LINEAR
+  [GL.TEXTURE_MAG_FILTER]: GL.LINEAR,
+  [GL.TEXTURE_WRAP_S]: GL.CLAMP_TO_EDGE,
```
Comment (Pessimistress) [inline modules/layers/src/bitmap-layer/bitmap-layer.js]: Can we split this into a separate PR?

**Comment by georgios-uber** [comment]
I think this is a great example to convert to WebVR using `VRAnimationLoop`.
@xintongxia / @Pessimistress What do you think?

**Comment by ibgreen** [comment]
> I think this is a great example to convert to WebVR using `VRAnimationLoop`.

I still think we should design things so that we can inject the `VRAnimationLoop` in the website without modifying it in the example source code.

This way we don't need to pull in an experimental class in our official source code, and we don't need to "choose" which examples are VR enabled.

**Comment by georgios-uber** [comment]
> > I think this is a great example to convert to WebVR using `VRAnimationLoop`.
> 
> I still think we should design things so that we can inject the `VRAnimationLoop` in the website without modifying it in the example source code.
> 
> This way we don't need to pull in an experimental class in our official source code, and we don't need to "choose" which examples are VR enabled.

How **inject** is going to work?

**Comment by ibgreen** [comment]
How inject is going to work?

Couple of options.
* We could change the API to make the "VRDisplay" into an AnimationLoop "plugin" rather than an AnimationLoop subclass. Maybe we offer a `Display` base class that can define viewports, matrices etc and make the AnimationLoop understand that.
* Or we could just mess around with imports and aliases when building the website.
* or we could just use the simple JS hack where a constructor can return something else:

```
class AnimationLoop {
  constructor({...}) {
	if (AnimationLoopClass) {
      return new AnimationLoopClass(...)
      // Some fiddling here to avoid going recurse...
    }
    ...
  }
````

File: website/src/components/demos/map-tile-demo.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+  static get allowMissingData() {
+    return true;
+  }
+
+  static get viewport() {
+    return INITIAL_VIEW_STATE;
+  }
+
+  static get mapStyle() {
+    return MAPBOX_STYLES.DARK;
```
Comment (Pessimistress) [inline website/src/components/demos/map-tile-demo.js]: Should use the blank map style. (see plot example)

File: website/src/components/demos/map-tile-demo.js [hunk 3]
```diff
@@ -0,0 +1,46 @@
+import React, {Component} from 'react';
+import {MAPBOX_STYLES} from '../../constants/defaults';
+import {App, INITIAL_VIEW_STATE} from 'website-examples/map-tile/app';
+
+export default class LineDemo extends Component {
+  static get parameters() {
+    return {
+      autoHighlight: {displayName: 'Enable autoHighlight', type: 'checkbox', value: true}
```
Comment (Pessimistress) [inline website/src/components/demos/map-tile-demo.js]: Call it "Highlight on hover"
